exercises/proverb/tests/proverb.rs

```diff
@@ -0,0 +1,52 @@
+extern crate proverb;
+
+use proverb::build_proverb;
+
+#[test]
+fn test_two_pieces() {
+    let input = vec!("nail", "shoe");
+    let expected = vec!("For want of a nail the shoe was lost.",
+                        "And all for the want of a nail.").join("\n");
+    assert_eq!(build_proverb(input), expected);
+}
+
+// Notice the change in the last line at three pieces.
+#[test]
+#[ignore]
+fn test_three_pieces() {
+    let input = vec!("nail", "shoe", "horse");
+    let expected = vec!("For want of a nail the shoe was lost.",
+                        "For want of a shoe the horse was lost.",
+                        "And all for the want of a horseshoe nail.").join("\n");
+    assert_eq!(build_proverb(input), expected);
+}
+
+#[test]
+#[ignore]
+fn test_one_piece() {
+    let input = vec!("nail");
+    let expected = vec!("And all for the want of a nail.").join("\n");
```

Comment:
This could be just a simple `"And all for the want of a nail.\n"` no need to allocate a Vector.